rpc/handlers.go

```diff
@@ -1271,6 +1280,10 @@ func (h *Handler) TransactionStatus(ctx context.Context, hash felt.Felt) (*Trans
 			Execution: receipt.ExecutionStatus,
 		}, nil
 	case ErrTxnHashNotFound:
+		if h.feederClient == nil {
+			return nil, ErrInternal.CloneWithData("no feeder client configured")
```

Comment:
Maybe we should just return `ErrTxnHashNotFound`?

Reply:
Makes sense